server/src/test/java/com/cloud/network/NetworkModelTest.java

```diff
@@ -90,6 +91,11 @@
     private static final long PHYSICAL_NETWORK_1_ID = 1L;
     private static final long PHYSICAL_NETWORK_2_ID = 2L;
 
+    private static final String IPV6_CIDR = "fd59:16ba:559b:243d::/64";
```

Comment:
Nowadays IPv6 networking support only `/64` CIDR. Added Unit tests covering this.